### ISSUE TITLE
table_describe.sql failing on sql2008R2, sql2014 databases

### ISSUE THREAD

**Issue by twelsh920** [TimelordUK/node-sqlserver-v8#101]
My stack trace looks like:
```
TypeError: meta.getSummary is not a function
    at new BulkTableOpMgr (C:\Code\fish-tools\node_modules\msnodesqlv8\lib\table.js:284:26)
    at meta (C:\Code\fish-tools\node_modules\msnodesqlv8\lib\table.js:508:25)
    at describeTable (C:\Code\fish-tools\node_modules\msnodesqlv8\lib\table.js:273:13)
    at metaResolver.getTableDefinition (C:\Code\fish-tools\node_modules\msnodesqlv8\lib\table.js:56:11)
    at Object.conn.query [as callback] (C:\Code\fish-tools\node_modules\msnodesqlv8\lib\meta.js:86:11)
    at onQueryRaw (C:\Code\fish-tools\node_modules\msnodesqlv8\lib\connection.js:103:20)
    at Immediate.setImmediate (C:\Code\fish-tools\node_modules\msnodesqlv8\lib\connection.js:113:13)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)

```

 Traced this down to an error in metaResolver.getTableDefinition:
`{ [Error: [Microsoft][SQL Server Native Client 11.0][SQL Server]Invalid column name 'generated_always_type'.] sqlstate: '42S22', code: 207 }`

Running the generated sql manually on my db through ssms returns:
Msg 207, Level 16, State 1, Line 40
Invalid column name 'generated_always_type'.
Msg 207, Level 16, State 1, Line 41
Invalid column name 'generated_always_type_desc'.
Msg 207, Level 16, State 1, Line 42
Invalid column name 'is_hidden'.

running my code on the same db, upgraded to SQL2016 instance runs without error.

EDIT:
fixing to npm package 0.7.2 also avoids this issue. Only 0.7.3 giving trouble

**Comment by TimelordUK** [comment]
https://github.com/TimelordUK/node-sqlserver-v8/blob/master/lib/queries/table_describe.sql

For above file can you try comment out 3 lines of select query as below 

/*
    c.generated_always_type,
    c.generated_always_type_desc,
    c.is_hidden,
*/

These were added for recent issue

Looks like needs to be conditional on server version

**Comment by TimelordUK** [comment]
this should now be fixed in 0.8.1

**Comment by TimelordUK** [comment]
This is fixed now so closing.
